ops/generate-subgraph.js

```diff
@@ -130,6 +132,12 @@ function combineFragments(fragments, isTemplate, addresses, missingAddresses) {
       return;
     }
 
+    if (isTemplate) {
+      // convert name to be version specific
+      const classVersion = version.replace(/\.|-/g, '_');
```

Comment:
how about generate a pre compiled mapping from for all specific version template ?
this will make the code more readable and maintainable.

Reply:
I'm not sure what you mean by this.

Reply:
see https://github.com/daostack/subgraph/pull/380/files#r342098970

Reply:
Will look into this.

Reply:
Looked into it and responded to your comment here, it's impossible to do this: https://github.com/daostack/subgraph/pull/380/files#r342098970